### ISSUE TITLE
fullscreen from maximized still doesn't work properly; title bar issue? cutout region issue?

### ISSUE THREAD

**Issue by 0xcaffebabe** [microsoft/terminal#4224]
# Environment

```none
Windows build number: Windows 10 1909 build 18363.535
Windows Terminal version (if applicable): 0.8.10091.0
shell:powershell
```

# Steps to reproduce

1.launch windows terminal
2.maximize windows terminal
2.press ctrl+enter into fullscreen mode

# Expected behavior
terminal should be a maximize state

# Actual behavior

but title bar has error state in fullscreen
![批注 2020-01-15 090830](https://user-images.githubusercontent.com/34188127/72396146-b2950280-3776-11ea-85a7-41eec0cab634.png)

**Comment by Tr4pSec** [comment]
Not seeing this issue in Windows 10 1909 build 18363.535 with Windows Terminal 0.8.10091.0

However I see this error in Windows 10 1903 build 18362.592 with Windows Terminal 0.8.10091.0

**Comment by zadjii-msft** [comment]
From @nafana in #4725

> 
> Should have tested this sooner, sorry! This is only reproducible if you have the "Automatically hide the taskbar in desktop mode" set to enabled in Windows settings.
> 
> To access this setting:
> 
>   1. Right-click on your taskbar
> 
>   2. Click Taskbar Settings

**Comment by drk-mtr** [comment]
I'm just going to put the words full screen here in the hope that it helps people find this. I stupidly hadn't thought to search fullscreen, so I filled out an issue template before realising it's a dup.

**Comment by zadjii-msft** [comment]
Hey so we've made a couple changes in this area (#4857, #4845), and no one on the team is seeing this anymore. We're thinking that this was fixed by a combo of those two PRs. If this actually isn't fixed in 0.10 (whenever that gets released), then feel free to post here and we'll re-open this one. 

Thanks all!
